### ISSUE TITLE
'clicked' Event not working 

### ISSUE THREAD

**Issue by agriguol** [blainesch/electron-notifier#24]
Hello! I am working electron-notifications but 'clicked' and 'swipeRight' not working.

notification.on('clicked', () => {
    console.log("Clicked");
    notification.close();
})

**Comment by Nantris** [comment]
From the documentation:

```
notification.on('buttonClicked', (text) => {
  if (text === 'Snooze') {
    // Snooze!
  }
  notification.close()
})
```

Note the use of `'buttonClicked'` as opposed to `'clicked'` you are using. Does that resolve it?

`swipeRight` should be `swipedRight` (with a 'd')

**Comment by cgerikj** [comment]
I've had the same problem on windows 7, 'clicked' doesn't seem to work but 'buttonClicked' does.

**Comment by agriguol** [comment]
The event buttonClicked works, but 'clicked' does not work. I need to make click in the notification without buttons.

**Comment by bayleedev** [comment]
@agriguol if you run the playbook does the click example work?

**Comment by agriguol** [comment]
@blainesch - 'clicked' event does not work with playbook. 

You must to correct the 'swipedEvent' event (with a 'd') in the playbook.

**Comment by bayleedev** [comment]
@agriguol fixed with 78955ab868989529731bdfc45d4075450fe9f2ce

Can you give me some details to help me reproduce this?

* Operating system
* node version
* any errors in the console you are getting

**Comment by agriguol** [comment]
Operating system : Windows 10 Home Version 1511
node version: v6.9.5
npm version: 3.10.10
any errors in the console you are getting : no errors

**Comment by bayleedev** [comment]
I think I have reproduced this on a VM, but I'm having issues getting devtools open on the notification. I'll play around with this later tonight. Thanks for reporting it!

**Comment by Nantris** [comment]
I am able to consistently get devTools with

Updating windowOptions to:
```
let windowOptions = {
      width: 400,
      height: this.notificationHeight + verticalSpace,
      x: size.width - 400,
      y: 0,  /* this is because we set the poper position when we pop it from the queue */
      frame: true,
      resizable: true,
      alwaysOnTop: true,
      skipTaskbar: true,
      webPreferences: {
        devTools: true,
      },
      focusable: true,
    };
```

and adding

`notificationWindow.openDevTools();` after `notificationWindow.loadURL()`

Not sure if you're having a general problem with devTools or just a specific one in that case. Hope this helps.

**Comment by afriedma** [comment]
I can re-produce it with palybook on windows 7

**Comment by afriedma** [comment]
Any plans on merging this fix?

**Comment by bayleedev** [comment]
@afriedma I have not found a way to fix this yet.

**Comment by Nantris** [comment]
I'm not clear on the issue here. Is it that we want to add a `'clicked'` listener to the entire notification?

**Comment by afriedma** [comment]
Yes, it used to work fine.  Any clues, I can help out if you needed.

**Comment by bayleedev** [comment]
@afriedma it's odd it would just stop, the behavior file hasn't been modified since it was created. The only thing that's changed recently was some css :/

**Comment by afriedma** [comment]
May be because you made the browser window same size as HTML div? Click is not being propogated?

Cheers, Alex

________________________________

**Comment by Nantris** [comment]
What element was the `clicked` listener attached to? Is it possible that another element is overlayed on top of the one you are trying to listen for clicks on?

For example, I noticed that the `title` has a `::before` element which covers it, which was preventing me from listening for clicks on that element.

**Comment by bayleedev** [comment]
On my VM I looked at every major version with a playbook (v0.1.0 and higher) and none of them appear to work on Windows with the click event, so this isn't a regression.

**Comment by afriedma** [comment]
I just did some testing too, @blainesch you are correct, click event doesn't work on windows.
